packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -544,22 +564,46 @@ class SliverChildListDelegate extends SliverChildDelegate {
   /// The widgets to display.
   final List<Widget> children;
 
+  Map<Key, int> _mapKeyToIndex;
+
+  int _findItemIndex(Key key){
+    // This function should only be called when any child is keep alive and has
```

Comment:
Is keep alive really a requirement? Just a key should be enough, right?

Reply:
I forgot to update this doc, nice catch!